pkgs/development/interpreters/ruby/default.nix

```diff
@@ -97,7 +97,7 @@ let
           (import ./patchsets.nix {
             inherit patchSet useRailsExpress ops;
             patchLevel = ver.patchLevel;
-          })."${ver.majMinTiny}";
+          })."${ver.majMinTiny}" or [];
```

Comment:
I think that, if we’re doing this, the existing rubies should be built using `mkRuby`, and their patches should be passed into it. (I’d be fine with getting rid of patchsets.nix and just inlining the patches here — there are a lot fewer patches than there used to be.)

Reply:
Should I not use the patches if `useRailsExpress` is false within the `mkDerivation`?